### PR TITLE
Notify car owner

### PR DESCRIPTION
**Pull request by namolnad**
@mhanlon Hey Matt, I've been working on this a bit (though still have some work to do to iron out kinks in auto layout—specifically around device orientation) and have gotten to a point where I think I want to spend a lot of my immediate focus on getting cloudKit up and running and then follow up with things like: 

- filling out the options for all of the buttons so there is a proper user flow
- making sure the other VC's are fully functioning
- adding options to attach pictures and other info, when appropriate
- being able to use the camera to input the license plate number

Before I fully move on, I have a question regarding the amount of logic in the parkingReporterVC. Is there too much in there? Because a lot what happens while the user is traveling through the user flow is very simple, I decided to add subViews to the existing VC and create a custom nav stack. Is it simple enough where this is appropriate? Or should I really be creating a new VC for each of these subsequent views? Thoughts?

Thanks!

### CODE REVIEW

File: Wilson/Wilson/ParkingReporterViewController.swift [hunk 1]
```diff
@@ -0,0 +1,616 @@
...
+    func hideNavBar()
+    {
+        self.navigationController?.navigationBar.barTintColor = UIColor(red: 0.341, green: 0.341, blue: 0.341, alpha: 1)
+        self.navigationController?.navigationBar.translucent = false
+        self.extendedLayoutIncludesOpaqueBars = true
+        self.navigationController?.navigationBar.shadowImage = UIImage()
+        self.navigationController?.navigationBar.setBackgroundImage(UIImage(), forBarMetrics: UIBarMetrics.Default)
+    }
+    
+    func addLabel()
```
Comment (mhanlon) [inline Wilson/Wilson/ParkingReporterViewController.swift]: Is this just placeholder code at the moment?

**Comment by mhanlon** [comment]
You're right, that class does get a bit big... and as you add more features on and the code branches down the damaged vehicle or break-in or one of the other paths, it'll just get bigger and bigger.

It might make sense to split them out and use the built-in view controller stack mechanism in UIKit... What you've done is good, and a great idea, I'm just trying to save you from re-writing the wheel! :)

And now, since you're using a storyboard there's a degree of view code chucked into the storyboard, itself, but you might want to create some custom views, perhaps to handle the buttons, or the license plate view. All these custom views can even be created (for the most part) in Interface Builder, just create the custom class in Xcode and dump a custom view out into your storyboard or a new xib/nib file and you can set up auto layout without the code...

This may simply be an architectural optimization that you make later, and not necessarily something you *need* to do right now.

File: Wilson/Wilson/ParkingReporterViewController.swift (hunk 1 above)
Comment (namolnad) [inline Wilson/Wilson/ParkingReporterViewController.swift]: Yes, the addLabel function was from when I was just first playing around with swift again. I'll probably be pulling that out soon. The hideNavBar function simply makes the navigation bar appear as though it's part of the rest of the view.

**Comment by namolnad** [comment]
Yeah, I think that's a great idea. Though I intended on working on cloudKit throughout the day today, I decided to do a bit of work refactoring my code per some of your suggestions. It's definitely not perfect yet, but it's a bit better. I recently pushed it to the origin on a new cloudKit branch (again, I had different intentions and then started obsessing with switching things around, hence the changes in the cloudKit branch) I'll be working on cloudKit now and will probably work on finalizing the refactoring/button auto layout later on.

**Pull request merged**